frontend/packages/knative-plugin/src/actions/add-subscription.ts

```diff
@@ -1,44 +1,28 @@
+import * as React from 'react';
```

Comment:
Recently we have decided that we should be moving away from namespace imports and instead import e.g. the hooks directly.
```suggestion
import { useMemo }  from 'react';
```